unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/Sections/PlacesAndEventsSection/PlacesAndEventsCardsFactory.cs

```diff
@@ -0,0 +1,97 @@
+using DCL;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using static HotScenesController;
+using Object = UnityEngine.Object;
+
+public static class PlacesAndEventsCardsFactory
+{
+    internal const string EVENT_CARD_MODAL_ID = "EventCard_Modal";
+    internal const string PLACE_CARD_MODAL_ID = "PlaceCard_Modal";
+
+    /// <summary>
+    /// Creates and configures a pool for cards.
+    /// </summary>
+    /// <param name="pool">Pool to configure.</param>
+    /// <param name="poolName">Name of the pool.</param>
+    /// <param name="cardPrefab">Card prefab to use by the pool.</param>
+    /// <param name="maxPrewarmCount">Max number of pre-created cards.</param>
+    public static void GetCardsPoolLazy(out Pool pool, string poolName, BaseComponentView cardPrefab, int maxPrewarmCount)
```

Comment:
What's the idea being having an out variable instead of returning a `Pool`?

Reply:
I don't know the original idea of that :) I just moved 2 copy of this method from `Places/EventUtils` classes to this Factory class. But it is really good point, I like it and I will convert it to return the `Pool`.

Reply:
fixed in https://github.com/decentraland/unity-renderer/pull/3917/commits/b2707034b81499b9c84465ab78436f0b0a5195c8